package.json

```diff
@@ -214,6 +214,14 @@
         }
       ]
     },
+    "breakpoints": [
```

Comment:
Is there a way to conditionally add this capability based on the server version? If not I guess we'll just have to live with it. The next stable version of Metals should come out soon

Reply:
Judging by the docs, I don't think so.

> we use the breakpoints contribution point to list the languages for which setting breakpoints will be enabled. Without this, it would not be possible to set breakpoints

Reply:
Fair enough